### DISCUSSION TITLE
How to get the vane settings into HA

### DISCUSSION THREAD

**Discussion by umtauscher** [echavet/MitsubishiCN105ESPHome#99] [General]
Hi,
thanks for the updates. I can now see the different vane settings in the device controls of homeassistant.
Is there any control that I can use to make them available to the dashboard?
I think I have tried all Lovelace and mushroom climate controls but none show this options. (horizontal and vertical vanes)
Am I missing something or is it not possible at he moment?
Cheers

**Comment by SixFive7** [comment]
I'd like to know that as well. Currently I have this weird setting in the card:

![image](https://github.com/echavet/MitsubishiCN105ESPHome/assets/6650122/01786990-6ed2-4776-ab85-c09233696cf2)

And these separate entities on the dashboard:

![image](https://github.com/echavet/MitsubishiCN105ESPHome/assets/6650122/adde8d97-4155-4b5e-95d1-327a7ff51616)

Would be nice if these could be integrated.

I did find some mods. [This one](https://github.com/geoffdavis/esphome-mitsubishiheatpump/issues/33#issuecomment-1197261844) seems to allow for renaming fan speeds and [this one](https://github.com/geoffdavis/esphome-mitsubishiheatpump/pull/109#issuecomment-2002038593) seems to include vane control in the card. But they are from another (and no longer active) project. Haven't got them to work in combination with this otherwise awesome repository yet.

**Comment by echavet** [comment]
> Hi,
> thanks for the updates. I can now see the different vane settings in the device controls of homeassistant.
> Is there any control that I can use to make them available to the dashboard?
> I think I have tried all Lovelace and mushroom climate controls but none show this options. (horizontal and vertical vanes)
> Am I missing something or is it not possible at he moment?
> Cheers 

Hi,
No it is not possible for the moment.
I didn't find a way to do that yet!!
